.github/workflows/test.yml

```diff
@@ -0,0 +1,26 @@
+name: CI
+on:
+  # Trigger the workflow on push or pull request,
+  # but push action is only for the feature branch
+  pull_request:
+    types: [ opened, synchronize, reopened ]
+  push:
+    branches-ignore:
+      - develop
+      - 'release/**'
+      - main
+
+jobs:
+  lint_and_test:
```

Comment:
I think we should update the job's name to match with the current steps. How about `run_template_script`? 💡

Reply:
@manh-t `test` is enough I think